instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/WrappedLambda.java

```diff
@@ -106,12 +107,19 @@ Method getRequestTargetMethod() {
                   if (a.getParameterCount() != b.getParameterCount()) {
                     return b.getParameterCount() - a.getParameterCount();
                   }
-                  if (isLastParameterContext(a.getParameters())) {
-                    return -1;
-                  } else if (isLastParameterContext(b.getParameters())) {
-                    return 1;
+                  boolean firstCtx = isLastParameterContext(a.getParameters());
+                  boolean secondCtx = isLastParameterContext(b.getParameters());
+                  // one of the methods has last param context ?
+                  if (firstCtx ^ secondCtx) {
```

Comment:
This logic has gotten pretty mysterious - would it be possible to add some example method signatures to the javadoc to explain it?

Reply:
Added :)